code/game/objects/items/melee/fireaxe.dm

```diff
@@ -52,3 +52,49 @@
 /obj/item/melee/axe/scrap/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=22, icon_wielded="[base_icon_state]_w")
+
+/*
+Blunt
+ */
+/obj/item/melee/axe/sledgehammer
```

Comment:
Is there any specific reason this is an axe subtype

Reply:
It was mostly laziness because it would've already inherited the window destroying thing. Do you want me to Not?

Reply:
No, I'm just wondering what the reasoning is

Reply:
ideally windowbreaking would be a component or something though

Reply:
code refactor issue (out of scope)